### PR TITLE
New Slovak cheat sheet

### PR DESCRIPTION
**Pull request by OndroNR**
**What does your Instant Answer do?**
Displays common phrases in Slovak language with appropriate English translations.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It shows common phrases just now without searching for good page or tracking.

**What is the data source for your Instant Answer? (Provide a link if possible)**
I translated [Czech cheat sheet](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/czech.json).

**Why did you choose this data source?**
I choose it as starting point for this cheat sheet as it provides common phrases and it is easy to translate as Slovak is similar language.

**Are there any other alternative (better) data sources?**
Can't think of any.

**What are some example queries that trigger this Instant Answer?**
`slovak cheat sheet`

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Tourists

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
I believe it was [this article](http://www.root.cz/zpravicky/duckduckgo-sutaz-ako-dostat-na-vianoce-cool-tricko/)

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot

-----
IA Page: https://duck.co/ia/view/slovak_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@OndroNR Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/slovak.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
+{
+   "id" : "slovak_cheat_sheet",
+   "name" : "Slovak Cheat Sheet",
+   "metadata" : {},
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/slovak.json]: @OndroNR is there no source for this info? We should at least reference a website where users can go to find more info. Most of the existing Language Cheat Sheets have been using Linguanaut: http://www.linguanaut.com/english_slovak.htm

Comment (OndroNR) [inline share/goodie/cheat_sheets/json/slovak.json]: It is translated from the czech cheat sheet. Thanks for the link, I will add it.

**Comment by OndroNR** [comment]
Just added metadata source link.

**Comment by tagawa** [comment]
Feedback from Twitter: The "How do you say \"X\" in Czech?" phrase should be "How do you say \"X\" in Slovak?"

File: share/goodie/cheat_sheets/json/slovak.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            "val" : "Som alergická.",
+            "key" : "I am allergic. (female)"
+         },
+         {
+            "val" : "Zopakujte to, prosím.",
+            "key" : "Repeat, please."
+         },
+         {
+            "val" : "Ako sa po slovensky povie \"X\"?",
+            "key" : "How do you say \"X\" in Czech?"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/slovak.json]: should be `in Slovak` I think?

Comment (OndroNR) [inline share/goodie/cheat_sheets/json/slovak.json]: Yes, you are right. I will fix it now.

**Comment by OndroNR** [comment]
Fixed. Thanks.

**Comment by moollaza** [comment]
@OndroNR this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@OndroNR congrats! It's live: https://duckduckgo.com/?q=Armenian+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**